src/web_interface/templates/base.html

```diff
@@ -103,17 +103,29 @@
                 <!-- Navbar Elements -->
                 <ul class="nav navbar-nav navbar-left" style="padding-right: 5px">
                     {% for caption, href, icon in navigation_bar %}
+                        {% if caption == "Database" %}
+                            <li class="dropdown {% if caption == active_page %}active{% endif  %}">
+                                <a class="dropdown-toggle" data-toggle="dropdown" href="#"><span class="glyphicon glyphicon-tasks"></span>&nbsp;Database<span class="caret"></span></a>
+                                <ul class="dropdown-menu">
+                                    <li><a href="/database/browse">Browse Database</a></li>
+                                    <li><a href="/database/search">Search Database</a></li>
+                                    <li><a href="/database/advanced_search">Advanced Search</a></li>
+                                    <li><a href="/database/binary_search">Binary Search</a></li>
+                                    <li><a href="/database/browse_compare">Browse Compare Results</a></li>
```

Comment:
We might rename and reorder these items for better understanding:
- Browse Firmware Analyses
- Browse Firmware Compares
- Basic Search
- Advanced Search
- Binary Pattern Search